amethyst_renderer/src/pass/shaders/fragment/shaded.glsl

```diff
@@ -64,16 +65,15 @@ void main() {
     vec4 ecolor = texture(emission, tex_coords(vertex.tex_coord, emission_offset.u_offset, emission_offset.v_offset));
     vec3 lighting = vec3(0.0);
     vec3 normal = normalize(vertex.normal);
-    for (uint i = 0u; i < point_light_count; i++) {
+    for (uint i = 0u; i < point_light_count && i < 1u; i++) {
```

Comment:
Only 1 light? :)

Reply:
Oh. How does this sneaky debugging change got here 😄